net-im/toxic/toxic-0.10.1-r1.ebuild

```diff
@@ -58,6 +60,8 @@ src_prepare() {
 	#This line changes the "lazy set if absent" assignment to a "lazy set" assignment.
 	#look below in src_configure to see how CFLAGS are passed to the makefile in USER_CFLAGS
 	sed -i -e 's/?=/=/g' Makefile || die "Unable to force CFLAGS and LDFLAGS"
+	#Fix incorrect include declarations for the macros NAME_MAX and PATH_MAX
+	eapply -p0 "${FILESDIR}/${P}-PATH_MAX-and-NAME_MAX.patch" || die "Unable to fix include statements"
```

Comment:
```
 * ERROR: net-im/toxic-0.10.1-r1::gentoo failed (prepare phase):
 *   patch -p1 -p0 failed with /var/tmp/portage/net-im/toxic-0.10.1-r1/files/toxic-0.10.1-PATH_MAX-and-NAME_MAX.patch
 * 
```

Reply:
I am unable to replicate the error regarding the patch for toxic-0.10.1-r1. I was able to replicate and fix (I hope) the linking with python error.